hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -398,22 +398,22 @@ export class MissingContainers extends React.Component<Record<string, object>, I
           <Tabs defaultActiveKey='1'>
             <Tabs.TabPane
               key='1'
-              tab={`Missing${(missingDataSource && missingDataSource.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
+              tab={`Missing${(missingDataSource?.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
               {generateTable(missingDataSource)}
             </Tabs.TabPane>
             <Tabs.TabPane
               key='2'
-              tab={`Under-Replicated${(underReplicatedDataSource && underReplicatedDataSource.length > 0) ? ` (${underReplicatedDataSource.length})` : ''}`}>
+              tab={`Under-Replicated${(underReplicatedDataSource?.length > 0) ? ` (${underReplicatedDataSource.length})` : ''}`}>
```

Comment:
```suggestion
              tab={`Under-Replicated${underReplicatedDataSource?.length ?? ''}`}>
```

Reply:
@devabhishekpal 
Done Changes in latest commit878b46ef6f3090e43a104f606bdf49ad890f0e4f